tests/lib/rules/no-useless-v-bind.js

```diff
@@ -136,6 +136,22 @@ tester.run('no-useless-v-bind', rule, {
         'Unexpected `v-bind` with a string literal value.',
         'Unexpected `v-bind` with a string literal value.'
       ]
+    },
+    {
+      code: `
+      <template>
+        <div :id="  'foo'  " />
+        <div :id='  "foo"  ' />
```

Comment:
Can you please add these test cases here? Are they handled correctly?

```pug
<div :id=" \"foo\" " />
<div :id="' \'foo\' " />
<div :id=" `foo` " />
<div :id=' `foo` ' />
```

Reply:
All of these test cases were fixed correctly, except the first one that seems invalid.

Reply:
Oh yeah, that was a typo on my side. I removed those test cases again, as this rule should not care about invalid v-bind values; there is [`vue/valid-v-bind`](https://eslint.vuejs.org/rules/valid-v-bind.html) for that.

Thanks for checking!